components/NewsletterSubscribe.js

```diff
@@ -1,24 +1,104 @@
+import { useState } from "react";
 import Button from "./buttons/Button";
 import Heading from "./typography/Heading";
 import Paragraph from "./typography/Paragraph";
+import Loader from "./Loader";
+import MultiSelect from "./form/MultiSelect";
 import { useTranslation } from "../lib/i18n";
 
-export default function NewsletterSubscribe ({
+export default function NewsletterSubscribe({
   className = 'p-8 text-center',
   dark = false,
-  title = 'Subscribe to our newsletter to receive news about AsyncAPI.',
-  subtitle = 'We respect your inbox. No spam, promise ✌️',
   type,
 }) {
 
-  const { t } = useTranslation('common'); 
+  const [email, setEmail] = useState('');
+  const [name, setName] = useState('');
+  const [status, setStatus] = useState("normal");
+  const [selectedSubscriptions, setSelectedSubscriptions] = useState([]);
```

Comment:
@akshatnema
Let's end the confusion. This is the state I was talking about in previous comments.